python/equistore-torch/tests/operations/add.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+import equistore.torch
+
+from .data import load_data
+
+
+def test_operation_as_python():
```

Comment:
instead of duplicating all the tests, could you use the same pattern as https://github.com/lab-cosmo/equistore/pull/327/commits/dd87c5bde341314e178f314a88bd3e00673559f9 or https://github.com/lab-cosmo/equistore/commit/b4969ed51d55a8a845acb4bad22ed1c86734ddc9#diff-2c35cd5d88b7f753ab0edbf63d88798ec6bab12cb0d6da020bbe19d427df434f?

Reply:
This way it is also easier to add more tests for the actual behavior of the code